ext/dav1d_android.sh

```diff
@@ -32,12 +30,11 @@ ABI_LIST=("armeabi-v7a" "arm64-v8a" "x86" "x86_64")
 ARCH_LIST=("arm" "aarch64" "x86" "x86_64")
 for i in "${!ABI_LIST[@]}"; do
   abi="${ABI_LIST[i]}"
-  mkdir "${abi}"
   cd "${abi}"
   PATH=$PATH:${android_bin} meson setup --default-library=static --buildtype release \
     --cross-file="../../package/crossfiles/${ARCH_LIST[i]}-android.meson" \
-    -Denable_tools=false -Denable_tests=false ../..
-  PATH=$PATH:${android_bin} ninja
+    -Denable_tools=false -Denable_tests=false "dav1d/build/${abi}" dav1d
+  PATH=$PATH:${android_bin} meson compile -C "dav1d/build/${abi}"
```

Comment:
Optional: Same here. I think it is fine to use the `ninja -C "dav1d/build/${abi}"` command.